droidlet/lowlevel/hello_robot/remote/obstacle_utils.py

```diff
@@ -57,6 +58,7 @@ def get_ground_plane(
 def is_obstacle(
     pcd,
     base_pos,
+    lidar=None,
```

Comment:
renaming this to `lidar_scan` might be useful. Wasn't clear to me if this was supposed to be a lidar python object (i.e. a handle to the lidar) or a lidar scan